libraries/core-react/src/Table/Cell.jsx

```diff
@@ -4,11 +4,28 @@ import styled from 'styled-components'
 import tableTokens from '@equinor/eds-tokens/components/table/table.json'
 import { typographyTemplate } from '../_common/templates'
 
-const { header, cell } = tableTokens
+const {
+  header: {
+    text: {
+      text: { typography: headerTypography, ...headerTextLevel2 },
+      ...headerTextLevel1
+    },
+  },
+  cell,
+} = tableTokens
 
 const variants = {
   header: {
-    text: header.text,
+    text: {
+      ...headerTextLevel1,
+      text: {
+        ...headerTextLevel2,
+        typography: {
+          ...headerTypography,
+          textAlign: 'left',
+        },
+      },
+    },
```

Comment:
We can simplify this further. We are already handling `textAlign` when exporting typography in `figma-broker`. 

Looks like we decided earlier to opt out `textAlign` prop if its value was left... 🤔

Simpler fix would be to update `figma-broker` transformer for typography [here](https://github.com/equinor/design-system/blob/4eba1e02c31d650a6052eb27da2f3ce7d4bf4458/apps/figma-broker/transformers/typography.js#L57) and then re-run `/create-component-tokens`